lib/addons/aws-node-termination-handler/index.ts

```diff
@@ -176,10 +184,13 @@ export class AwsNodeTerminationHandlerAddOn extends HelmAddOn {
         return {
             enableSqsTerminationDraining: true,
             queueURL: queue.queueUrl,
+            awsRegion: karpenter ? cluster.stack.region: '',
```

Comment:
@youngjeong46 where you able to test the queue mode?

Reply:
yes - both Queue and IMDS were tested using v0.16.3 (the latest before the native interruption handling) per Karpenter best practices/official doc.